web/packages/teleport/src/services/version/unsupported.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { ApiError } from '../api/parseError';
+
+export function withUnsupportedLabelFeatureErrorConversion(
+  err: unknown
+): never {
+  if (err instanceof ApiError && err.response.status === 404) {
+    if (err.proxyVersion && err.proxyVersion.string) {
```

Comment:
Having `proxyVersion` will be probably useful in the future, but for this particular feature we won't be able to show it. Returning proxy version will land in the same version as the V2 endpoint, so we either won't show the error or show the more generic version from below.

Reply:
yeah that's the plan, to make it fallback to the generic messaging for the time being

Reply:
So we can remove the more specific version as we will never show it.

Reply:
oh that's true! i'll remove it